packages/component-library/src/components/form/sw-text-field/sw-text-field.stories.ts

```diff
@@ -26,7 +26,7 @@ export default {
     <div>
       <sw-text-field
         v-bind="args"
-        :modelValue="currentValue"
+        v-model="currentValue"
         @inheritance-restore="inheritanceRestoreWrapper"
         @inheritance-remove="inheritanceRemoveWrapper"
         @change="onChange">
```

Comment:
If the `v-model` is used here than we can get rid of the 'change` event listener line 32.

Reply:
The change event listener is needed for showing the event inside the Storybook events